form-validation.js

```diff
@@ -7,29 +7,26 @@ const messageMobile = document.querySelector('#messageEmail-mobile');
 const nameInput = document.querySelector('.name');
 const emailInput = document.querySelector('.email');
 const messageInput = document.querySelector('.message');
-const rawData = localStorage.getItem('cachedFormData');
+const savedData = JSON.parse(localStorage.getItem('formData')) || {};
 
-const formobject = {
-  name: '',
-  email: '',
-  message: '',
-};
-
-if (rawData) {
-  const formobject = JSON.parse(rawData);
-  emailInput.value = formobject.email;
-  nameInput.value = formobject.name;
-  messageInput.value = formobject.message;
+if (savedData) {
+  nameInput.value = savedData.name;
+  emailInput.value = savedData.email;
+  messageInput.value = savedData.message;
 }
 
-const dataStore = (event) => {
-  const element = event.target;
-  formobject[element.name] = element.value;
-  localStorage.setItem('cachedFormData', JSON.stringify(formobject));
+const saveData = () => {
+  const formData = {
+    name: nameInput.value,
+    email: emailInput.value,
+    message: messageInput.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
 };
-nameInput.addEventListener('change', dataStore);
-emailInput.addEventListener('change', dataStore);
-messageInput.addEventListener('change', dataStore);
+
+nameInput.addEventListener('input', saveData);
+emailInput.addEventListener('input', saveData);
+messageInput.addEventListener('input', saveData);
```

Comment:
- [ ] What a brilliant change you've made here! :clap: :clap: It is just that the app seems buggy at the moment. The local storage functionality works very fine in smaller screens (mobiles) but the functionality that helps your user to preserve or persevere data in their browsers is not working for bigger screens (desktop users), and this isn't good for user experience. Please kindly make the changes, which also checks with the suggestion from the previous code reviewer. 
- Kindly watch this [voiceover video](https://watch.screencastify.com/v/i5U0REidIMzR3ja6t5MN) of a tour I had on your portfolio page for more clues about this malfunctioning. :pray: